docs/src/proposals/gasless-evm.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Gasless EVM
+---
+
+## Problem
+
+Applications targeting other blockchains use audited smart contracts
+implemented in Solidity. Running that same application targeting Solana
+requires migrating the smart contract to Rust and the Sealevel parallel
+programming model. Creating the same smart contract in Solana requires
+mechanical transformations of high-level Solidity constructs to lower-level
+Rust equivalents. The process is tedious and allows for human error, requiring
+a second audit of the new code. Automating the process would save time and
+alleviate the need for an audit.
+
+## Proposed Solution
+
+Add an EVM bytecode loader. Map Solidity contract storage to Solana account
+data. Map runtime data to Solana sysvars. Require transactions to specify which
+accounts they may access, such that Sealevel may execute independent contracts
+in parallel. Instead of requiring "gas", map EVM bytecode to BPF so that
+contracts are subject to the BPF instruction limit. Support Solidity method
+calls from arbitrary Solana programs and executing Solana instructions from
+Solidity methods.
+
+### Developer Experience
+
+```bash
+solc --bin sourceFile.sol > lib-hello.bin
```

Comment:
This isn't the command we want, but all the right information is in that file. Instead, we want a binary with the binaries of each contract and some header to look up a contract's bytecode.

Reply:
constructor, runtime, and abi info